### DISCUSSION TITLE
UTM M1 crash - Windows 10

### DISCUSSION THREAD

**Discussion by AlessandroCordella** [utmapp/UTM#3706] [UTM Help (macOS)] [answered]
Last April I downloaded the Windows 10 Arm VHDX file (it seems not available anymore, only Windows 11 Arm VHDX). I tried running it with Parallels and it worked fine.

Now I need to create a Win10 virtual machine using UTM. I converted the VHDX file to a qcow2 file using the following qemu command:
`qemu-img convert -p -O qcow2 Windows10_InsiderPreview_Client_ARM64_en-us_21354.VHDX Windows10.qcow2`

Then I created a VM using UTM with the following settings:

![immagine](https://user-images.githubusercontent.com/44475440/156008001-609f8af8-3ba0-4149-b74b-fd6b434cab10.png)
![immagine](https://user-images.githubusercontent.com/44475440/156008105-567d2e63-8cb4-47ef-a373-4186913c4ec1.png)

but during the boot process I get this error INACCESSIBLE_BOOT_DEVICE:
![immagine](https://user-images.githubusercontent.com/44475440/156008181-6047d2fc-1a33-46d1-bf4d-53bcced22b31.png)

Then, I tried to convert the VHDX file to a qcow2 file from a Windows machine using StarWind V2V Converter (the qcow2 converted file was different that the one created with qemu). I create a VM in UTM using the same configuration, but I got the same error during the boot process.

I read what is written here https://github.com/utmapp/UTM/issues/2646, but I have already set the hardisk as a NVMe.

What configuration am I doing wrong? I report that I tried the same process using the Windows 11 VHDX file and the qcow2 generated using StarWind V2V Converter, and the Windows 11 machine is running correctly in UTM in my MAC M1.

**Comment by osy** [comment]
Please provide the following:

```
**Configuration**
* UTM Version: 
* OS Version: 
* Intel or Apple Silicon? 
```

**Comment by AlessandroCordella** [comment]
UTM 3.0.4 (46)
macOS Monterey 12.2.1
Apple Silicon

**Comment by osy** [comment]
Did you use the wizard? If so you should use the VHDX directly and not a converted qcow2 since it will convert it for you.

**Comment by AlessandroCordella** [comment]
I tried using the wizard and selecting directly the VHDX, but I got the same error

**Comment by osy** [comment, marked as answer]
Oh I just realized you said “ Windows10_InsiderPreview_Client_ARM64_en-us_21354.VHDX Windows10.qcow2” that’s too old and doesn’t support M1 at all. You need 21390 or higher. I recommend using https://mac.getutm.app/gallery/windows-10

**Reply by AlessandroCordella** [reply]
I solved using [Windows 10 Insider Preview 10.0.21390.2025 (co_release) arm64](https://uupdump.net/selectlang.php?id=7811e991-621e-4b0c-a066-ee0d4b53eacf). Thank you!

**Comment by dopeytree** [comment]
Does this work anymore on virtualised setting?

**Comment by ilanbiala** [comment]
I also have a VM where using IDE drive type works but switching to NVMe causes the same INACCESSIBLE_BOOT_DEVICE error and BSOD.

UTM 4.7.4 (115)
macOS 15.7.1
Apple Silicon (M2 Max)
